src/Driver/RemoteClient.php

```diff
@@ -691,7 +691,7 @@ private function export(callable $upgrade, Request $request, Response $response,
                 return;
             }
 
-            $this->logger->error($exception);
+            $this->logger->error($exception->getMessage(), ['exception' => $exception]);
```

Comment:
I have no idea what this is about... can you write some message for this please?

Reply:
`Unexpected exception during socket upgrade, closing connection`

Reply:
```suggestion
            $this->logger->error('Unexpected exception during socket upgrade, closing connection.', ['exception' => $exception]);
```